content/influxdb/cloud-dedicated/admin/query-system-data.md

```diff
@@ -119,6 +128,112 @@ Replace the following:
 When prompted, enter `y` to acknowledge the potential impact querying system
 tables may have on your cluster.
 
+### Optimize queries to reduce impact to your cluster
+
+Querying InfluxDB v3 system tables may impact the performance of your
+{{< product-name omit=" Clustered" >}} cluster.
+As you write data to a cluster, the number of partitions and Parquet files
+can increase to a point that impacts system table performance.
+Queries that took milliseconds with fewer files and partitions might take 10
+seconds or longer as files and partitions increase.
+
+Use the following filters to optimize your system table queries and reduce the impact on your
+cluster's performance.
+
+##### Filter by table name
+
+Filter by `table_name` when querying `system.tables`, `system.partitions`, or `system.compactor`. 
+
+```sql
+WHERE table_name = 'TABLE_NAME' (string)
+```
+
+##### Filter by partition key 
+
+Filter by `partition_key` when querying `system.partitions` or `system.compactor`.
+
+```sql
+WHERE partition_key = 'PARTITION_KEY (string)
+```
```

Comment:
~~~suggestion
When querying the `system.partitions` or `system.compactor` tables, use the `WHERE` clause to
filter by `partition_key` .

```sql
SELECT * FROM system.partitions WHERE partition_key = 'PARTITION_KEY'
```
~~~